content/en/synthetics/guide/browser-tests-totp.md

```diff
@@ -0,0 +1,48 @@
+---
+title: TOTPs For Multi-Factor Authentication (MFA) in Browser test
+kind: guide
+further_reading:
+   - link: 'https://www.datadoghq.com/blog/mfa-synthetic-testing-datadog/'
+     tag: 'Blog'
+     text: 'Introducing multi-factor authentication in Datadog Synthetic tests'
+   - link: 'synthetics/settings/?tab=specifyvalue#global-variables'
+     tag: 'Documentation'
+     text: 'Learn more about global variables'
+   - link: 'synthetics/browser_tests/actions/'
+     tag: 'Documentation'
+     text: 'Learn more about browser test steps'
+ 
+---
+
```

Comment:
```suggestion
<div class="alert alert-warning">
RBAC restrict access to global variables is in beta. To request access, contact <a href="https://docs.datadoghq.com/help/">Datadog support</a>.</div>  

```

Reply:
I would not do that as TOTP are usable for everyone and only the Permission Tab needs the special action.
I am concerned that it could scare our customers

Reply:
Would it be possible to put this warning inside the element of the list?

Reply:
Ok, sounds good!